src/components-examples/material/button-toggle/button-toggle-forms/button-toggle-forms-example.html

```diff
@@ -0,0 +1,6 @@
+<mat-button-toggle-group name="fontStyle" [formControl]="fontStyle" aria-label="Font Style">
```

Comment:
I don't think this needs a `name` since it isn't inside a `form`.

Reply:
fixed